website/content/docs/enterprise/pkcs11-provider/oracle-tde.mdx

```diff
@@ -106,28 +106,41 @@ We need to configure the Vault PKCS#11 provider.
 
 1. If you want to view the Vault logs (helpful when trying to find error messages), you can specify the `VAULT_LOG_FILE` (default is stdout) and `VAULT_LOG_LEVEL` (default is `INFO`). We'd recommend setting `VAULT_LOG_FILE` to something like `/tmp/vault.log` or `/var/log/vault.log`. Other useful log levels are `WARN` (quieter) and `TRACE` (very verbose, could possibly contain sensitive information, like raw network packets).
 
-## Enable TDE
+## Enable TDE 
 
 The only remaining step is to setup Oracle TDE for an external HSM using shared library, `libvault-pkcs11.so`.
 These steps are not specific to Vault, other than requiring the shared library, HCL configuration, and certificates be present.
 TDE is complex, but an example way to enable it is:
 
-1. Create or update the `sqlnet.ora` (usually in `$ORACLE_BASE/oradata/dbconfig/$SID/sqlnet.ora`) configuration file to use an HSM encryption wallet by adding the following line:
+1. Open a `sqlplus` session into the root container (or switch into it with `ALTER SESSION SET CONTAINER = CDB$ROOT;`).
 
+1. Set WALLET_ROOT and TDE_CONFIGURATION parameters on the Oracle database.
+
+   ~> **Note**: The directory is a root wallet directory and is only required for the purposes of setting `TDE_CONFIGURATION`. More on these parameters in the [Oracle databse Documentation](https://docs.oracle.com/en/database/oracle/oracle-database/19/refrn/TDE_CONFIGURATION.html).
```

Comment:
```suggestion
<Note>
  The wallet root directory is only used to set the TDE configuration parameter.
  To learn more about the wallet parameters refer to the
  <a href="https://docs.oracle.com/en/database/oracle/oracle-database/19/refrn/TDE_CONFIGURATION.html">Oracle TDE documentation</a>.
</Note>
```

Style issue: use new tag notation for asides. (There is currently a bug where markdown doesn't parse properly with the new notation, so links should use HTML for now)

Reply:
Thank you. I believe I committed the suggestion.?